src/utils/redis-search-stack/build-search-query.js

```diff
@@ -96,6 +120,31 @@ const searchParamBuilder = {
     const name = buildParamName(FIELD_PREFIX, propName, ParamSuffix.match);
     return [name, expr.match];
   },
+  any: (prop, expr) => {
+    const propName = normalizePropName(prop);
+    const params = [];
+
+    if (prop.endsWith('_tag')) {
+      expr.any.forEach((item, index) => {
+        const name = buildParamName(FIELD_PREFIX, propName, ParamSuffix.any, index);
+        params.push(name, quotedString(item));
+      });
+
+      return params;
+    }
+
+    expr.any.forEach((param, index) => {
+      const subparams = tokenize(param).map(
```

Comment:
Separation into single operation and OR-concatenation make the code more reusable